js/src/components/SnippetPreviewSection.js

```diff
@@ -17,6 +17,22 @@ const Section = styled( StyledSection )`
 	}
 `;
 
+/**
+ * Function to map the editor data to data for the preview.
+ *
+ * @param {Object} mappedData The default mappedData object.
+ * @param {string} mappedData.title The processed title.
+ * @param {string} mappedData.url The baseUrl with the slug.
```

Comment:
This is the only param without 'processed' in the description. Has the url not been processed/mapped before?